### DISCUSSION TITLE
Where do we see the error logs generated by the wiki js?

### DISCUSSION THREAD

**Discussion by suniltorase** [requarks/wiki#3119] [Help / Questions]
Hello,

We have an error in the system. Some users tried to logs in but getting no success and apparently getting an authentication error message. Since the auth details are correct they should not display this message. We are trying to trace the root cause of this issue. 
 
We are running the wiki.js using "node server" and trying to see the error logs generated by the wiki js in the logs directory. 
In the config.yml file, the "logLevel" parameter is set to "info" (we also tried it with type debug, error, etc...). But it fails to produce any error logs except the info.

Basically, we are looking for detailed and all types of error logging. 
Any suggestions??

Thank you

**Comment by mattpr** [comment]
It sounds like you found the logs, so the question I guess is whether this type of AUTH failure should be expected to show up in the application logs.  Whether the developer added a debug statement for AUTH failures or not...you would need to just check the code.

From my end, I wouldn't consider AUTH failures to be application errors unless something strange/unexpected happens.  If a user tries to login and fails, we consider this a user error rather than application error that should be logged on the server.

If you wanted login success/fail audit logs for security purposes, well that is a different thing than normal application error logs.

Options:

- If you can get with the user to reproduce the issue, I would persist the browser dev tools network view and see what is happening.  Is this oauth error or just a simple password failure.  You might get a clue on the client side.  Perhaps the error message to the user could be more helpful.
- If you can't sit with the user or get enough information client-side... well this is open source javascript... so you can just open up any file on your server in vim, add some logging statements at the critical points you care about and restart the `node server` process.
